Fix bug error on importing Events in python 3.11
as in https://github.com/pytorch/ignite/pull/2907
Author: Cintia Willemyns (Vrije Universiteit Brussel)

--- ignite/engine/events.py.orig	2024-01-03 16:37:16.972475097 +0100
+++ ignite/engine/events.py	2024-01-03 16:39:03.703040949 +0100
@@ -248,8 +248,10 @@
                 # ...
     """
 
-    pass
-
+    def __new__(cls, value: str) -> "EventEnum":
+        obj = CallableEventWithFilter.__new__(cls)
+        obj._value_ = value
+        return obj
 
 class Events(EventEnum):
     """Events that are fired by the :class:`~ignite.engine.engine.Engine` during execution. Built-in events:
